src/smt/smt_quantifier.cpp

```diff
@@ -170,15 +170,21 @@ namespace smt {
         void display_stats(std::ostream & out, quantifier * q) {
             quantifier_stat * s     = get_stat(q);
             unsigned num_instances  = s->get_num_instances();
+            unsigned num_instances_simplify_true = s->get_num_instances_simplify_true();
+            unsigned num_instances_checker_sat  = s->get_num_instances_checker_sat();
             unsigned max_generation = s->get_max_generation();
             float max_cost          = s->get_max_cost();
-            if (num_instances > 0) {
+            if (num_instances > 0 or num_instances_simplify_true>0 or num_instances_checker_sat>0) {
```

Comment:
For compatibility: use "||" instead of "or".
The pull request currently does not build because of this.

Reply:
Note that this is currently blocking the PR

Reply:
done.